src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      state.categories.push('Under Construction');
```

Comment:
- [ ] Since we will not be working with the categories for the time being, we do not want to add the phrase `Under Construction` to the array every time this action is dispatched. Kindly update it in such a way that it returns the string `Under Construction`. 🤝️

**Example**
```
state.categories = action.payload === "Under construction" ? "Under construction" : state.categories
```